web/src/list_widget.js

```diff
@@ -284,15 +295,16 @@ export function create($container, list, opts) {
         // the sorting function is either the function or string that calls the
         // function to sort the list by. The prop is used for generic functions
         // that can be called to sort with a particular prop.
```

Comment:
This comment needs to be updated. Because we have removed `prop`, and that we expect a different format for `sorting_function` if it is a `string`.

Reply:
Oh yeah, nice catch!